e2e/local-artifact-mirror_test.go

```diff
@@ -34,9 +34,9 @@ func TestLocalArtifactMirror(t *testing.T) {
 		{"systemctl", "stop", "local-artifact-mirror"},
 		{"systemctl", "start", "local-artifact-mirror"},
 		{"systemctl", "status", "local-artifact-mirror"},
-		{"curl", "-o", "/tmp/kubectl-test", "127.0.0.1:50000/bin/kubectl"},
-		{"chmod", "755", "/tmp/kubectl-test"},
-		{"/tmp/kubectl-test", "version", "--client"},
+		{"curl", "-o", "/tmp/kubectl", "127.0.0.1:50000/bin/kubectl"},
```

Comment:
Since kubectl is now a symlink `http.FileServer` returns the linked file with is k0s. Everything will still work it just happens that it is the actual k0s binary rather than a symlink.

Another option is to change kubectl to a script that execs the k0s binary.